cpp/test/Slice/utf8BOM/test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) ZeroC, Inc.
+
+import os
+import shutil
+
+from Util import ClientTestCase, SliceTranslator, TestSuite
+
+
+class SliceUtf8BomTestCase(ClientTestCase):
+    def runClientSide(self, current):
+        test_file_name = "Test.ice"
+        slice2cpp = SliceTranslator("slice2cpp")
+
+        # Temporary output directory for 'slice2cpp to use.
+        outdir = "{0}/tmp".format(os.getcwd())
+        if os.path.exists(outdir):
+            shutil.rmtree(outdir)
+        os.mkdir(outdir)
+
+        try:
+            # Make sure the BOM marker wasn't accidentally deleted by over-zealous reformatting.
+            current.write("Checking for UTF8-BOM marker... ")
+            with open(test_file_name, 'rb') as test_file:
+                BOM = test_file.read(3)
+                if (BOM[0] != 0xEF or BOM[1] != 0xBB or BOM[2] != 0xBF):
```

Comment:
Another option is something like:
```suggestion
                if BOM[:3] != b'\xEF\xBB\xBF':
```

Reply:
Fixed!